nematode.js

```diff
@@ -342,7 +466,7 @@ class Nematode {
             this.sprite.alpha = Math.max(0, this.sprite.alpha - delta * 0.5);
         }
 
-        // Remove the nematode from the world after 10 seconds
+        // Remove the nematode from the world after 14 seconds
```

Comment:
Probably should just use a setTimeout(() => destroy nematode, 14000) when the nematode dies

Reply:
Also, who knows exactly when the setTimeout() function will call the lambda. If it calls it at the wrong time, there might be an error or something.